pkg/setting/setting.go

```diff
@@ -768,7 +768,7 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	// read dashboard settings
 	dashboards := iniFile.Section("dashboards")
 	DashboardVersionsToKeep = dashboards.Key("versions_to_keep").MustInt(20)
-	MinRefreshInterval, err = valueAsString(dashboards, "min_refresh_interval", "")
+	MinRefreshInterval, err = valueAsString(dashboards, "min_refresh_interval", "5s")
```

Comment:
May want to change this so that `MinRefreshInterval` becomes a `time.Duration`?

Reply:
Sounds good. But that could be another PR. :)

Reply:
Yeah, noticed it quite a bit of changes so probably some time later.